docs/modules/ROOT/pages/Thirdparty_Plugins.adoc

```diff
@@ -831,7 +831,7 @@ You can also manually trigger this with Mill by doing the following:
 [source, shell script]
 ----
 
-mill --import ivy:io.chris-kipp::mill-scip::0.2.2 io.kipp.mill.scip.Scip/generate
+mill --import ivy:io.chris-kipp::mill-scip::0.3.4 io.kipp.mill.scip.Scip/generate
```

Comment:
Looks good. Ideally, we can use a asciidoc variable and fill it from the `build.sc`, to keep it up-to-date.

Alternatively, for your convenience, we can add a TODO to not forget the idea:

```suggestion
// TODO: Provide the scip version via variable, see https://github.com/com-lihaoyi/mill/pull/2621
mill --import ivy:io.chris-kipp::mill-scip::0.3.4 io.kipp.mill.scip.Scip/generate
```

Reply:
did the recommended approach